### ISSUE TITLE
Less chatty in logs please

### ISSUE THREAD

**Issue by drnic** [cloudfoundry/dns-release#9]
Whilst tailing all my instance logs, the screen is being constantly overwritten with bosh-dns logs. Could we update/refresh/poll/etc much less regularly so as to reduce logs? Or any other ideas?

```
==> /var/vcap/sys/log/bosh-dns/bosh_dns.stdout.log <==
[File System] 2017/11/29 05:08:58 DEBUG - Stat '/var/vcap/instance/dns/records.json'
[File System] 2017/11/29 05:08:59 DEBUG - Stat '/var/vcap/instance/dns/records.json'
[File System] 2017/11/29 05:09:00 DEBUG - Stat '/var/vcap/instance/dns/records.json'

==> /var/vcap/sys/log/bosh-dns/bosh_dns_resolvconf.stdout.log <==
[File System] 2017/11/29 05:09:00 DEBUG - Reading file /etc/resolv.conf
[File System] 2017/11/29 05:09:00 DEBUG - Read content
********************
# This file was automatically updated by bosh-dns
nameserver 169.254.0.2

nameserver 8.8.8.8
nameserver 10.10.0.2
search us-east-2.compute.internal

********************
```

**Comment by dpb587** [comment]
+1 it’s unneccessarily noisy; and has noticeably increased the resource requirements of centralized logging in my environments

**Comment by cppforlife** [comment]
several stories in progress to address this  (dns4 label).

**Comment by cppforlife** [comment]
fixes went into master and 0.1.1
